requirements/base.txt

```diff
@@ -104,4 +104,11 @@ ed25519
 tweepy
 oogway==0.7.0
 icalendar==4.0.7
+duniterpy
+aiohttp >= 3.6.3
+jsonschema
+pyPEG2
+base58
+libnacl
+pyaes
```

Comment:
can the versions be fixed?

Reply:
@developerfred

Reply:
this is def needed

Reply:
@developerfred can we fix the version numbers?

Reply:
e5757bebd51fafe997dab8e831070232844daaee